File: src/main/java/pt/joaomneto/titancompanion/adventurecreation/impl/TPOPAdventureCreation.java
package pt.joaomneto.titancompanion.adventurecreation.impl;


import java.io.BufferedWriter;
import java.io.IOException;

public class TPOPAdventureCreation extends TFODAdventureCreation{


    @Override
    protected void storeAdventureSpecificValuesInFile(BufferedWriter bw)
            throws IOException {

        super.storeAdventureSpecificValuesInFile(bw);

        bw.write("copper=0\n");
    }



}

File: src/main/java/pt/joaomneto/titancompanion/adventurecreation/impl/TPOPAdventureCreation.kt
package pt.joaomneto.titancompanion.adventurecreation.impl

import java.io.BufferedWriter
import java.io.IOException

class TPOPAdventureCreation : TFODAdventureCreation() {

    @Throws(IOException::class)
    override fun storeAdventureSpecificValuesInFile(bw: BufferedWriter) {

        super.storeAdventureSpecificValuesInFile(bw)

        bw.write("copper=0\n")
    }
}
